### ISSUE TITLE
General: Mention SecureDrop

### ISSUE THREAD

**Issue by tommycollison** [freedomofpress/encryption-works#38]
*Encryption Works* 1.0 doesn't mention SecureDrop once -- I agree with Runa's [suggestion](https://github.com/freedomofpress/encryption-works/issues/7) that we should, but instead of a section, I just included a mention in the "about" section currently tucked in at the bottom (although maybe should be a prelude of sorts?) 

In any case, this is the wording I've been toying with:

# About Freedom of the Press Foundation

> “A cantankerous press, an obstinate press, a ubiquitous press must be suffered by those in authority in order to preserve the even greater values of freedom of expression and the right of the people to know.” \
—Judge Murray Gurfein, Pentagon Papers case, June 17, 1971

The **Freedom of the Press Foundation** is a 501(c)3 non-profit organization dedicated to helping support and defend public-interest journalism focused on exposing mismanagement, corruption, and law-breaking in government. We accept tax-deductible donations to a variety of journalism organizations that push for transparency and accountability, and we work to preserve and strengthen the rights guaranteed to the press under the First Amendment. We manage [SecureDrop](https://securedrop.org), an open-source whistleblower submission system that media organizations can use to securely accept documents from and communicate with anonymous sources. It was originally created by the late Aaron Swartz. Click [here](https://freedom.press/securedrop/directory) to see a full list of news organizations which have implemented SecureDrop.

**Comment by garrettr** [comment]
@tommycollison I think this is a good start. However, it seems to me that we might also want to mention SecureDrop in the document itself, and not just as a note in the footer. Do you think that would be a worthwhile addition?

**Comment by tommycollison** [comment]
@garrettr Agreed, but I'm not sure straight away where that place would be. Trevor's introduction, maybe.
